docs/circuits.ipynb

```diff
@@ -579,6 +579,137 @@
     "Especially useful is dropping the last moment (which are often just measurements): `circuit[:-1]`, or reversing a circuit: `circuit[::-1]`.\n"
```

Comment:
<p>I think we should also add a section to explain circuit_op.repeat_until and <code>circuit_op.mapped_circuit</code>.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5399/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
`repeat_until` requires measurement and classical control, neither of which are discussed in this high-level doc. Happy to add `mapped_circuit`, though.

Reply:
Done.